gatsby-node.js

```diff
@@ -93,8 +93,8 @@ exports.createPages = async ({ graphql, actions }) => {
     const curLangPosts = posts.filter(
       (p) => p.node.fields.langKey === post.node.fields.langKey
     )
-    const previous = curLangPosts[index + 1]?.node
-    const next = curLangPosts[index - 1]?.node
+    const previous = curLangPosts[index + 1] && curLangPosts[index + 1].node
```

Comment:
for some reason I couldn't build the app locally with the `curLangPosts[index + 1]?.node` notation

Reply:
interesting ... apparently gatsby doesn't yet support optional chaining, so I'm not sure why this builds on my machine. we can just not use this notation for now then